README.md

```diff
@@ -94,39 +124,68 @@ This section describes the OAuth flow steps to get the access token required to
 ```
   - In your project’s build.gradle file under buildscript, make the following changes
 ```
-			buildscript {
-				repositories {
-					mavenCentral()
-					jcenter()
-					google()
-				}
-
-				dependencies {
-					classpath 'com.android.tools.build:gradle:3.1.4'
-				}
-			}
-			allprojects {
-				repositories {
-					jcenter()
-					google()
-				}
-			}
+            buildscript {
+                repositories {
+                    mavenCentral()
+                    jcenter()
+                    google()
+                }
+
+                dependencies {
+                    classpath 'com.android.tools.build:gradle:3.1.4'
+                }
+            }
+            allprojects {
+                repositories {
+                    jcenter()
+                    google()
+                }
+            }
 ```
   - In your app module’s build.gradle file, add the following line under dependencies
 ```
-		api project(':roam')
-		implementation ("com.firstdata.clovergo:remote-pay-android-go-connector:3.3.1.3@aar") {
-			transitive = true
-		}
+        api project(':roam')
+        implementation ("com.firstdata.clovergo:remote-pay-android-go-connector:3.3.1.3@aar") {
+            transitive = true
+        }
+```
+  - In your `GoStartupActivity.java`, set your **accessToken**, **apiKey**, and **secret** in the following code block for Sandbox:
```

Comment:
Need to change variable names:
accessToken => demoAccessToken
apiKey => goApiKey
secret => goSecret